### DISCUSSION TITLE
Layouts avalible in Congo

### DISCUSSION THREAD

**Discussion by MikiVanousek** [jpanther/congo#128] [General]
From what I understand, there are only these layouts available for the pages on my Congo website:

- for the index 
  - `author`
  - `page`
  - `profile`
- for any other page:
  - `list`, automatically applied to branches
  - `single`, automatically applied to leafs

The last 2 I was able to override with the `layout` parameter of my front mater. But `layout: page` does not seem to work. Is there a way for me to use the `page` layout on other pages than the index homepage?

Generally, the information about layouts was a bit confusing to me. I feel like it is a weakness in otherwise outstanding documentation. If @jpanther thinks it's a good idea, I might submit a PR, where I extend the `Home Layout` section of the docs to a more general `Layouts` section, that includes some addition information on other layouts available.

**Comment by jpanther** [comment]
Thanks for the feedback about the documentation. When you say it's confusing, are you referring to the [Content Examples](https://jpanther.github.io/congo/docs/content-examples/) section?

You are correct that there's only `list` and `single` templates in the theme (ignoring taxonomies). What is it about the `single` layout that isn't meeting your needs?

**Reply by MikiVanousek** [reply]
The `Content Examples` section indeed answers all my questions about layouts. Perhaps a link `You can read about layouts of other pages in the 'Content Examples' section` to the `Homepage layout` section would make the docs easier to navigate? But it's a detail, the docs are amazing.

What is it about the `single` layout does not meet my needs? A lot of my pages are not blog posts, but just landing pages. I introduce my podcast, my app, etc. If the `single` template is applied, they show author, date, pagination and reading time by default. This is the desired behavior for blog posts, but I do not want this for my landing pages. 

Right now, I just manually disable these for each page separately in the front matter.
```
showAuthor: false
showDate: false
showReadingTime: false
showPagination: false
```
First, this is award, as there are multiple pages I need to apply this to separately. Second of all, hiding all the information I want to have for my blog posts creates an awkward empty space below my title.

This is a landing page with `layout: single` automatically assigned to it.
![image](https://user-images.githubusercontent.com/33402194/154429380-91995e37-ad1e-4709-96e1-31da27e4e3e2.png)

The same page, if I override the layout with `layout: list`. Notice how the icons are exactly below the title. This is the desired behavior.
![image](https://user-images.githubusercontent.com/33402194/156154311-9c7f924d-5758-4c3d-b168-74e2840ae8c8.png).

If I were able to use the `page` layout that can be applied to my home page, it would resolve all my problems.
